templates/compose/signoz.yaml

```
@@ -0,0 +1,624 @@
+# documentation: https://signoz.io/docs/introduction/
+# slogan: An observability platform native to OpenTelemetry with logs, traces and metrics.
+# tags: telemetry, server, applications, interface, logs, monitoring, traces, metrics
+# logo: svgs/signoz.svg
+# port: 8080
+
+services:
+  init-clickhouse:
+    image: clickhouse/clickhouse-server:25.5.6-alpine
+    command:
+      - bash
+      - -c
+      - |
+        version="v0.0.1"
+        node_os=$$(uname -s | tr '[:upper:]' '[:lower:]')
+        node_arch=$$(uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/)
+        echo "Fetching histogram-binary for $${node_os}/$${node_arch}"
+        cd /tmp
+        wget -O histogram-quantile.tar.gz "https://github.com/SigNoz/signoz/releases/download/histogram-quantile%2F$${version}/histogram-quantile_$${node_os}_$${node_arch}.tar.gz"
+        tar -xvzf histogram-quantile.tar.gz
+        mkdir -p /var/lib/clickhouse/user_scripts/histogramQuantile
+        mv histogram-quantile /var/lib/clickhouse/user_scripts/histogramQuantile
+    restart: on-failure
+    logging:
+      options:
+        max-size: 50m
+        max-file: "3"
+
+  zookeeper:
+    image: signoz/zookeeper:latest
```

There is no `latest` tag for this service on docker hub (https://hub.docker.com/r/signoz/zookeeper/tags?name=latest) as @Mortalife mentioned. 

Please change it to an valid version tag

```suggestion
  zookeeper:
    image: signoz/zookeeper:3.9.3
```

---

Suggestion only: You could also make the versions env variables so that patching existing versions can be a simple `Environment Variables > Developer View > Paste` rather than having to manually trawl through the compose to update.